server/utils/firebase_algolia_server_code/functions/algoliaFunctions.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * Create's an algolia record from a firebase entity snapshot.
+ * @param {function(string): !Promise} algoliaOperation a save function
+ * @param {Object} snapshot a json string
+ * @return {!Promise}
+ */
+function addEntityToIndex(algoliaOperation, snapshot) {
```

Comment:
nit: better name could be `createRecordFromEntity`

Reply:
Good suggestion, will change to that.

Reply:
There is a vocab note at the top of the index.js file. Index is what algolia calls their database. I agree with your points below though about renaming.